index.html

```diff
@@ -6985,6 +6985,9 @@ <h2>Changes since Candidate Release of 12 December 2019</h2>
           as it had accidentally been set to run for every entry in the object, rather than
           after all entries had been expanded.
           This is in response to <a href="https://github.com/w3c/json-ld-api/issues/262">Issue 262</a>.</li>
+        <li>Move step <a href="#alg-expand-null-value">15.3</a> down one, and return
```

Comment:
Incomplete line?  Maybe should be this:
```suggestion
        <li>Move step <a href="#alg-expand-null-value">15.3</a> down one, and return null.
```